apps/etterlatte-behandling/src/main/kotlin/behandling/migrering/MigreringService.kt

```diff
@@ -47,6 +49,12 @@ class MigreringService(
                     finnEllerOpprettSak(request)
                 }
             inTransaction {
+                val behandlinger =
+                    behandlingService.hentBehandlingerForSak(sak.id).filter { it.status != BehandlingStatus.AVBRUTT }
+                if (behandlinger.isNotEmpty() && behandlinger.any { !it.erAutomatiskGjenoppretting() }) {
```

Comment:
Hm, kva skjer viss vi har fleire løpande automatiske gjenopprettingar? Slit litt med å sjå for meg at det skjer, men

Reply:
Det er vel ikke mulig å ha flere løpende samtidig? Men tenker du på om det allerede er en iverksatt som var automatisk gjenoppretting?

Reply:
Trur forresten det er unødvendig med isNotEmpty-sjekken her, den vil bli dekka av any.